drivers/soc/aspeed/Makefile

```diff
@@ -7,3 +7,5 @@ obj-$(CONFIG_ASPEED_SOCINFO)		+= aspeed-socinfo.o
 obj-$(CONFIG_ASPEED_SBC)		+= aspeed-sbc.o
 obj-$(CONFIG_ASPEED_XDMA)		+= aspeed-xdma.o
 obj-$(CONFIG_ASPEED_UDMA)		+= aspeed-udma.o
+obj-$(CONFIG_AST2600_ESPI)              += ast2600-espi.o
+obj-$(CONFIG_AST2700_ESPI)              += ast2700-espi.o
```

Comment:
can we take AST2700 espi later? let's add only gpio on g7 in defconfig and disable espi until conflict is resolved

Reply:
Agreed will disable 2700 Defconfig.